zenodo/base/views.py

```diff
@@ -55,6 +55,7 @@ def register_menu_items():
 
     item = current_menu.submenu('main.browse')
     item.register('', _('Browse'), order=3)
+    item._active_when = lambda: request.endpoint.startswith("search.collection")
```

Comment:
@lnielsen, could you take a look ? I think accessing the `private` attribute in this way is really bad practise. Is there any better way to do it?